lib/debt_management_center/vbs/request.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+module DebtManagementCenter
+  module VBS
```

Comment:
Can you provide specs for this new class? Also, should this be moved into a module? I know that we are trying to move new code into modules (if possible) to help clarify code ownership as we break out microservices from vets-api

Reply:
@LindseySaari Oh yes, I'll get some specs for this! 

> should this be moved into a module?

Good question, I _would_ like to move this to a module, especially as this service's features are growing it's become coupled with one of our other applications (medical copays) and would like to refactor in the near future. The work for that refactor would be larger than originally anticipated for this ticket and PR however so I was going to generate a new one.

Reply:
Ok no problem! That sounds like a great plan. Maybe just note in the PR description that the plan is to move this into a module eventually.